Print to PDF (#35)
opengisch/QField-docs: en/project-management/print-to-pdf.rst

Print to PDF
==================================

It's possible to export the maps and other items to a PDF document.

If the project has at least one print layout defined, they are available over the menu.

.. container:: clearer text-center

  .. figure:: ../images/print_menu.jpg
     :width: 500px
     :alt: Print to PDF menu option

After selecting the print layout, the PDF document is created and stored in the devices documents directory.

By choosing the PDF reader app, the document can be opened directly.

.. container:: clearer text-center

  .. figure:: ../images/print_open.jpg
     :width: 500px
     :alt: Open PDF directly

Done.

.. container:: clearer text-center

  .. figure:: ../images/print_document.jpg
     :width: 500px
     :alt: PDF document

> .. note::  The devices documents directory is :code:`USER>/<APPNAME>/Documents` :code:`QField` means in the end something like :code:`////storage/emulated/0/Documents/QField`

Reviewer: ```suggestion
.. note::  The devices documents directory is :code:`USER>/<APPNAME>/Documents`. For :code:`QField` this is usually a path like :code:`////storage/emulated/0/Documents/QField`
```
